lib/cli/interactive-setup/aws-credentials.js

```diff
@@ -86,33 +193,208 @@ const steps = {
       )}\n`
     );
   },
+  handleProviderCreation: async ({ configuration: { org: orgName } }) => {
+    const providersUrl = `https://app.serverless.com/${orgName}/settings/providers?source=cli&providerId=new&provider=aws`;
+    openBrowser(providersUrl);
+    process.stdout.write('Waiting for creation of AWS Access Role provider...\n');
+
+    process.stdout.write(
+      '\nTo learn more about providers, visit: http://slss.io/add-providers-dashboard\n'
+    );
+
+    let onEvent;
+    let showSkipPromptTimeout;
+
+    const p = new Promise((resolve) => {
+      let inquirerPrompt;
+
+      const timeoutDuration = 1000 * 60; // 1 minute
```

Comment:
I was thinking about bringing it to even higher number. The thing here is that the provider setup flow takes some time to complete - you have to go to dashboard, understand what providers are all about, go to AWS account (maybe even create a new one if needed), create a CF stack with IAM Role and only then go back to CLI. It will definitely take more time than 1 minute. Do you think it should be lower?

Reply:
I think if it's too long, it might be that this feature doesn't serve anyone.

It should serve users which decided to abort at any point. It can be that user opened dashboard and decided to abort for some reason.

So, question is, why actually we're upholding this opt-out option. Is it harmful to show it to users which actually follow the setup process?

Reply:
> It should serve users which decided to abort at any point. It can be that user opened dashboard and decided to abort for some reason.

That is the goal, however, we don't want to show it too soon, to let people try to fill out the providers flow on the Dashboard side and only then allow them to opt-out and continue with steps. If it's too short we might have people opening dashboard and almost immediately giving up on setting the provider.

Reply:
> If it's too short we might have people opening dashboard and almost immediately giving up on setting the provider.
@medikoo 

I think there's slight problem with message. It should not encourage to _press enter_. Maybe it'll be better to put it as:

```
[If you approached an issue when setting up a provider, you may press ENTER to skip this step]
```

Having that, showing it even after 15s should be fine. What do you think?